src/OrchardCore/OrchardCore/Modules/ModularTenantRouterMiddleware.cs

```diff
@@ -36,9 +36,9 @@ public class ModularTenantRouterMiddleware
 
         public async Task Invoke(HttpContext httpContext)
         {
-            if (_logger.IsEnabled(LogLevel.Information))
+            if (_logger.IsEnabled(LogLevel.Debug))
             {
-                _logger.LogInformation("Begin Routing Request");
```

Comment:
there are too many useless records in the log when the info level is turned on

Reply:
I suggest to keep the changes as the title mentioned, so please revert the log changes and you can start a new PR

Thanks